prow/spyglass/viewers/buildlog/buildlog_viewer.go

```diff
@@ -67,12 +67,18 @@ type LogViewData struct {
 	Operation string `json:"operation,omitempty"`
 }
 
+// LogLines are broken up into SubLines so error terms can be highlighted.
+type SubLine struct {
+	Highlighted bool
+	Text        string
+}
+
 // LogLine represents a line displayed in the LogArtifactView.
 type LogLine struct {
 	Number      int
 	Highlighted bool
 	Skip        bool
-	Text        string
+	SubLines    []SubLine
```

Comment:
I like it 👍

Reply:
I'm pretty sure I was somehow subconsciously compelled to make the field SubLine.Text 🤔